source/how-tos/support_ticket.rst

```diff
@@ -0,0 +1,132 @@
+.. _support_ticket_guide:
+
+Support Ticket
```

Comment:
This section doesn't have anything on `rt` support. Can you add that as well?

Reply:
Will do.